lib/need_for_type/states/game.rb

```diff
@@ -54,15 +54,9 @@ def handle_init_game
     end
 
     def handle_start_game
```

Comment:
I think we can merge the `init_game` sub state into the `start_game` sub state because there is no more need to have an `init_game_state`.